swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0):
+        super().__init__(category, condition)
```

Comment:
This works fine! But one thing to consider is that this class `Clothing` should always be "clothing" right? Why give the user the power to accidentally or intentionally change that as a parameter?

instead, let's take the decisions away from them completely:

```suggestion
    def __init__(self, condition=0):
        super().__init__("clothing", condition)
```